src/dinitcheck.cc

```diff
@@ -88,21 +88,21 @@ int main(int argc, char **argv)
                         service_dir_opts.set_specified_service_dir(argv[i]);
                     }
                     else {
-                        cerr << "dinitcheck: '--services-dir' (-d) requires an argument" << endl;
```

Comment:
We need to have "endl"?

Reply:
The `endl` there looks superfluous. It might be needed elsewhere, if the buffer needs to be flushed.